### ISSUE TITLE
Issues with Colour Output (Wording/Layout/Newlines/Etc)

### ISSUE THREAD

**Issue by WaLLy3K** [pi-hole/pi-hole#1548]
**In raising this issue, I confirm the following:**

- [x] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md).
- [x] The issue I am reporting can be *replicated*
- [x] The issue I am reporting isn't a duplicate (see [FAQs](https://github.com/pi-hole/pi-hole/wiki/FAQs), [closed issues](https://github.com/pi-hole/pi-hole/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aclosed%20), and [open issues](https://github.com/pi-hole/pi-hole/issues)).

**How familiar are you with the codebase?:**

9

---

## Please use this ticket to report any issues you see with the new colour output seen in the `development` branch. This will help us keep track of the issues, and allow us to resolve them quicker!

When reporting an issue, the most important things to do are to provide a screenshot, and preferably, how the issue can be reproduced. You are welcome (and *encouraged*) to any issue with `pihole` output, no matter how trivial it might be!

Issues may also be reported in the [relevant Reddit thread](https://www.reddit.com/r/pihole/comments/6iqjto/see_something_say_something_new_development/).

#### Things we're looking for:
* Wording that could be phrased clearer, or made consistent with other parts of `pihole`
* Text that flows onto a new line (Longer than an 80 char width Terminal window)
* Text that gets jammed onto one line [such as this](https://github.com/pi-hole/pi-hole/pull/1471#issuecomment-310227304)
* Text that is not indented, that should be (with the exception of output provided by `git` such as ["Your branch is up-to-date with 'origin/development'"](https://i.imgur.com/j9Ydqjx.png))
* Anything relevant that we might have missed

#### Known Issues:
* "Update repo" text in `basic-install.sh` does not create a new line when printing which file was removed, [as seen here](https://github.com/pi-hole/pi-hole/pull/1471#issuecomment-310227304)
* "[✓] Pi-hole core" is printed after "Web interface" in `piholeCheckout.sh`, despite the confirmation message being "[i] Already up-to-date" [as seen here](http://i.imgur.com/j9Ydqjx.png)
* As per above issue, Interface in "Web interface" and Core in "Pi-hole core" should be capitalised

**Comment by Mcat12** [comment]
Also, `core` in `Pi-hole core` should be capitalized.

**Comment by technicalpyro** [comment]
i've noticed (not sure if intended) that the checkmarks work but its just an i if its not a checkmark ?

**Comment by WaLLy3K** [comment]
As you can see with @PromoFaux's [example GIF](https://cloud.githubusercontent.com/assets/1998970/26037771/990f08ae-38f2-11e7-8345-e55b78c752a8.gif), everything starts as a message portraying information ([i]), and then if something needs to portray a confirmation of a commands success/failure, it'll get a tick or a cross. Other messages such as "Repair option selected" just keep the [i].

Hopefully that makes sense?

**Comment by technicalpyro** [comment]
yeah it does going to go through all my usual command combos in the next couple hours and see if i can find anything just slightly aggravated that A I thought it was an issue and B didnt check one last time before mentioning lol

**Comment by Th3M3** [comment]
I think I have found one more:

(this is the info message after changing _Upstream DNS Server_)
![message](https://user-images.githubusercontent.com/28313514/27741666-274f8890-5db6-11e7-8e6e-e00ef996903d.jpg)

**Comment by DL6ER** [comment]
@PromoFaux

**Comment by Th3M3** [comment]
@WaLLy3K @PromoFaux 
I found one more :sweat_smile:

This occurs on `query adlists`-site, when searching for a domain which is in the ___wildcard___ blocklist:

![wildcard](https://user-images.githubusercontent.com/28313514/27772270-01726d68-5f5f-11e7-9aca-d649d2144f35.JPG)

**Comment by WaLLy3K** [comment]
> [i] Enabling lighttpd service to start on reboot...   Error: Unable to complete update, please contact support

**Comment by Th3M3** [comment]
when trying to add an invalid domain (`12345678`) to blacklist/whitelist in Web Console,
the red box shows `[✗] 12345678 is not a valid argument or domain name!`.
But when adding `invalid%.com`, the error message is only  `invalid%.com is not a valid domain`

**Comment by Th3M3** [comment]
@WaLLy3K @PromoFaux
In `/etc/pihole/install.log`, there are lines like
<pre>
  [i] Installing sudoer file...
<b>^[[K  [^[[1;32m✓^[[0m] Installing sudoer file</b>

  [i] Installing latest Cron script...
<b>^[[K  [^[[1;32m✓^[[0m] Installing latest Cron script</b>

  [i] Installing latest logrotate script...
<b>^[[K  [^[[1;32m✓^[[0m] Installing latest logrotate script</b>
</pre>

**Comment by PromoFaux** [comment]
```
fap@fappotron:~$ cat /etc/pihole/install.log
  [✓] Installing scripts from /etc/.pihole

  [i] Installing configs from /etc/.pihole...
  [i] Existing dnsmasq.conf found... it is not a Pi-hole file, leaving alone!
  [✓] Copying 01-pihole.conf to /etc/dnsmasq.d/01-pihole.conf

  [i] Creating log and changing owner to dnsmasq... log already exists!

  [i] Installing blocking page...
  [i] Installing index.php... detected index.php, not overwriting
  [i] Installing index.js... detected index.js, not overwriting
  [i] Installing blockingpage.css... detected blockingpage.css, not overwriting

  [✓] Installing sudoer file

  [✓] Installing latest Cron script

  [✓] Installing latest logrotate script

  [i] FTL Checks...
  [✓] Detected x86_64 architecture
  [i] Checking for existing FTL binary...
  [i] Latest FTL Binary already installed (v2.9.4). Confirming Checksum...
  [i] Checksum correct. No need to download!

```

**Comment by jacobsalmela** [comment]
Resolved in v3.2 (#1549, #1575)
